From ab82f014d81d19a9602edff208abe981a6c26eb9 Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Wed, 23 Nov 2022 18:46:48 -0500
Subject: [PATCH 1866/1936] Partial revert of plugin instrument loading commit

Based on comments by Stazed on #213, this commit should resolve a
regression in loading patches under a LV2 plugin host.
---
 src/Plugin/ZynAddSubFX/ZynAddSubFX.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/Plugin/ZynAddSubFX/ZynAddSubFX.cpp b/src/Plugin/ZynAddSubFX/ZynAddSubFX.cpp
index aa38e6e5..b0ae324b 100644
--- a/src/Plugin/ZynAddSubFX/ZynAddSubFX.cpp
+++ b/src/Plugin/ZynAddSubFX/ZynAddSubFX.cpp
@@ -284,8 +284,11 @@ protected:
     */
     void loadProgram(uint32_t index) override
     {
+        setState(nullptr, defaultState);
+        /*
         //in plugin mode, only the first part is useful
         middleware->pendingSetProgram(0, index);
+        */
     }
 
    /* --------------------------------------------------------------------------------------------------------
-- 
2.47.0

